### ISSUE TITLE
Missing example for RESUME in orders.md

### ISSUE THREAD

**Issue by TimothyAlexisVass** [on-api/on-api#113]
https://github.com/on-api/on-api/blob/master/2.4.0/spec/orders.md

The POST section presents examples for Activate, Deactivate, Suspend and Modify and also needs one for _Resume a service_.

**Comment by jonjacs** [comment]
Seems like the first of two modify examples should have been a resume example.
![screenshot_1451](https://github.com/on-api/on-api/assets/30898404/cd4914fd-92d3-47a0-a45d-026c95cc450a)

**Comment by TimothyAlexisVass** [comment]
Yeah that makes sense.

Also, the description is a little bit confusing for modify and suspend.
Now it seems like they only respond with `"state": "DONE_SUCCESS"` without any `"state": "RECEIVED"`.
Perhaps there should be some clarification included in the doc?

**Comment by teemufrosen** [comment]
I agree. The examples are inadequate in general, as orders have multiple states. Will look into making them clearer in upcoming versions, as well as adding diagrams for asynchronous flows.
